handlers/mapping_handlers.go

```diff
@@ -247,6 +240,18 @@ func (env *Env) PatchMappingHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if userID != memberID && *sessionMember.Pending {
+		errMsg := "Cannot modify other users in conversation while invitation is pending"
+		log.Println(errMsg)
+		http.Error(w, errMsg, http.StatusForbidden)
+		return
+	} else if *sessionMember.Pending && (reqMember.Role != "" || reqMember.Nickname != nil) {
```

Comment:
Users can't modify their own `Role` anyway, so that check is kinda redundant, but for the `Nickname`, I figured it doesn't really make sense to accept an invitation (i.e., set `Pending` to `false`) and change your `Nickname` in the same request.

Reply:
Makes sense.